templates/pages/entry.html

```diff
@@ -88,7 +100,13 @@
                         </div>
                     </div>
                     <!-- Creation / Update Time -->
-                    <p><small class="text-muted">Last updated on 15/06/20</small></p>
+                    <p><small class="text-muted">{% if entry.updated_on %}
```

Comment:
Once again, `small` is not the appropriate tag for this. You are using it in order to make something look smaller, but that is not the purpose of HTML. CSS should be used for cosmetic alterations.